packages/flutter_lints/pubspec.yaml

```diff
@@ -0,0 +1,13 @@
+name: flutter_lints
+description: Recommended lints for Flutter apps, packages, and plugins
```

Comment:
End with a period?

=> 

    Recommended lints for Flutter apps, packages, and plugins.